### ISSUE TITLE
Results are combined across multiple transaction calls

### ISSUE THREAD

**Issue by heydemo** [oskarer/websql-promisified#1]
I would expect the promise returned by transaction to only return Web SQL results that originated in that transaction - however - If I call two transactions in close succession the 2nd transaction promise includes the transaction from the first.

Not sure if this is intended or a bug, but it makes it difficult to have unrelated code that shares a database - Any suggestions?

Example:

``` javascript
var openDatabase = require('websql');
var webSqlPromise = require('websql-promisified').default;
var db = webSqlPromise(openDatabase('test-db', "1.0", "Test Database", 50000));

db.transaction(function(tx) {
  tx.executeSql('select "hello" as greeting'); 
})
.then(function(results) {
  console.log('first transaction results');
  console.log(results);
  // returns first transaction result
});

db.transaction(function(tx) {
  tx.executeSql('CREATE TABLE IF NOT EXISTS Person (name TEXT, age Integer)');
})
.then(function(results) {
  console.log('second transaction results');
  console.log(results);
  //return first AND second transaction result
});

```

**Comment by oskarer** [comment]
Great find! It's because the transactions in your example are executed in parallel and the results array is shared between all transactions. It should definitely not work like this and I'm working on a fix. Meanwhile you can run the second transaction after then first one is finished to get the correct behavior.

**Comment by oskarer** [comment]
Fixed in 1.0.4. Thanks for reporting this @heydemo!

**Comment by heydemo** [comment]
Awesome - thanks for quick response!
